include/sm64.h

```diff
@@ -706,19 +706,20 @@ enum MarioActionFlags {
 // group 0x1C0: custom actions
 // 0x1C0 - 0x1FF
 
-/*
- this input mask is unused by the controller,
- but END_DEMO is used internally to signal
- the demo to end. This button cannot
- be pressed normally by a controller.
-*/
-#define END_DEMO       (1 << 7)
+
+/**
+ * This input mask is unused by the controller,
+ * but END_DEMO is used internally to signal
+ * the demo to end. This button cannot be
+ * pressed normally by an N64 controller.
+ */
+#define INPUT_END_DEMO  X_BUTTON
```

Comment:
why is this being set to one of the gamecube buttons? seems a bit weird if its meant to be unused by everything except the demo inputs